src/components/app-content-renderer/styles/panels.scss

```diff
@@ -33,6 +33,13 @@
     padding: var(--pf-global--spacer--xl) var(--pf-global--spacer--lg) var(--pf-global--spacer--lg) var(--pf-global--spacer--lg);
     overflow-x: auto;
 
+    .ins-c-skeleton {
+      animation-duration: 3s;
+      &.ins-m-dark {
+        background: linear-gradient(to right, rgba(60,60,60,.2) 10%, rgba(72,72,72,.4) 18%, rgba(60,60,60,.2) 33%);
```

Comment:
I think it would be reasonable to backport this to PF as well. I've opened an issue[1] on their side so it can be implemented in PF directly, but in the meantime let's use this one!

[1] https://github.com/patternfly/patternfly/issues/4012